ios/FluentUI.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,23 +1,25 @@
 {
-  "pins" : [
-    {
-      "identity" : "appcenter-sdk-apple",
-      "kind" : "remoteSourceControl",
-      "location" : "https://github.com/microsoft/appcenter-sdk-apple.git",
-      "state" : {
-        "revision" : "25f64229373de97ff3920941cd52203193e5d8be",
-        "version" : "4.3.0"
+  "object": {
```

Comment:
I've also had this file changing a bunch of me recently. Not sure if we need to update it, but if we do it should probably be in a different PR.

Reply:
I can remove it from this pr at the very least